src/devices/Hcsr04/Hcsr04.cs

```diff
@@ -32,11 +33,13 @@ public class Hcsr04 : IDisposable
         /// <param name="gpioController">GPIO controller related with the pins</param>
         /// <param name="triggerPin">Trigger pulse input.</param>
         /// <param name="echoPin">Trigger pulse output.</param>
-        public Hcsr04(GpioController gpioController, int triggerPin, int echoPin)
+        /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
+        public Hcsr04(GpioController gpioController, int triggerPin, int echoPin, bool shouldDispose = true)
         {
             _echo = echoPin;
             _trigger = triggerPin;
             _controller = gpioController;
+            _shouldDispose = shouldDispose;
```

Comment:
you don't necessarily have to fix this, but I do wanted to point out that in a few bindings like this one,

Reply:
Disregard this comment.